puente-data-exporter/lambdas/data-exporter/index.py

```diff
@@ -121,4 +120,11 @@ def err_msg(msg: str) -> dict:
     }
 
 if __name__ == '__main__':
-    handler()
+    event = {
+       "queryStringParameters": {
+            "surveyingOrganization": "Ryans Well",
+            "specifier": "HistoryEnvironmentalHealth"
+       } 
+    }
+    context = None
+    handler(event, context)
```

Comment:
@awalsh272 we should change this back right?

Reply:
Fixed in new commit